api/src/handlers/list_validators.rs

```diff
@@ -137,7 +137,7 @@ pub async fn get_validators(
         OrderField::Stake => |a: &&ValidatorRecord| a.activated_stake,
         OrderField::MndeVotes => |a: &&ValidatorRecord| a.mnde_votes.unwrap_or(0.into()),
         OrderField::Credits => |a: &&ValidatorRecord| Decimal::from(a.credits),
-        OrderField::MarinadeScore => |a: &&ValidatorRecord| Decimal::from(a.marinade_score),
+        OrderField::MarinadeScore => |a: &&ValidatorRecord| Decimal::from((a.score.unwrap_or(0.0) * 10000.0) as u64),
```

Comment:
nitpick: I saw that "10000" is used in multiple places, perhaps we should set as a const and + give it a name

Reply:
Done